yuca/ca/common.py

```diff
@@ -215,6 +215,15 @@ def update_universe(self, identity, neighborhoods):
 
         return identity + neighborhoods
 
+    def update_helper(self, universe):
```

Comment:
Facilitates cloning update behavior when training NCAs on CA behavior by providing a function that takes only the current grid state.